packages/quantic/force-app/main/default/classes/HeadlessController.cls

```diff
@@ -0,0 +1,6 @@
+global with sharing class HeadlessController {
+    @AuraEnabled(continuation=true cacheable=false)
+    public static String getHeadlessConfiguration() {
+        return CustomTokenProvider.getHeadlessConfiguration();
```

Comment:
So the "default" package references the "custom-token" package? What happens if you just do a simple sfdx deploy ?

Reply:
You probably meant to use the "SimpleTokenProvider" here maybe?

Reply:
```suggestion
        return SampleTokenProvider.getHeadlessConfiguration();
```